util/election.go

```diff
@@ -67,6 +78,11 @@ func (ne *NoopElection) Close(ctx context.Context) error {
 	return nil
 }
 
+// GetCurrentMaster returns the string "It's you!"
+func (ne *NoopElection) GetCurrentMaster(ctx context.Context) (string, error) {
+	return "It's you!", nil
```

Comment:
Should this return `ne.instanceID` instead?

Reply:
Sure, done :)